setup.cfg

```diff
@@ -52,6 +52,7 @@ tests =
     pylint==2.15.0
     pylint-plugin-utils>=0.6
     mypy==0.981
+    py==1.11.0
```

Comment:
Hi @francisquintallauzon , sorry I am not fully understanding why this change is needed. How are you running the tests?

Reply:
@daavoo, tests were not running with the current config on my MacOS without this addition to setup.cfg.  I did not try to reproduce the issue with the Github Action's environment though and it might just be happening on my computer. 

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/1488870/217118475-dc8a6753-aa5f-49a1-bf3d-eb0576fe942f.png">